private/buf/cmd/buf/command/breaking/breaking.go

```diff
@@ -217,6 +218,22 @@ func run(
 	defer func() {
 		retErr = errors.Join(retErr, wasmRuntime.Close(ctx))
 	}()
+	// We add all check configs (both lint and breaking) as related configs to check if plugins
+	// have rules configured.
+	allCheckConfigs := append(
+		slicesext.Map(
+			imageWithConfigs,
+			func(imageWithConfig bufctl.ImageWithConfig) bufconfig.CheckConfig {
+				return imageWithConfig.LintConfig()
+			},
+		),
+		slicesext.Map(
+			imageWithConfigs,
+			func(imageWithConfig bufctl.ImageWithConfig) bufconfig.CheckConfig {
+				return imageWithConfig.BreakingConfig()
+			},
+		)...,
+	)
```

Comment:
Maybe a for loop is easier here as it would avoid needing to build the two lists? Or we could maybe add an `AllConfigs() []bufconfig.CheckConfig` method to `ImageWithConfig`.

Reply:
I think this makes sense, pushed up a change for this :)